SOURCES/0002-backport.patch

```diff
@@ -0,0 +1,360 @@
+--- a/igc.h	2022-09-23 12:17:01.000000000 +0000
++++ b/igc.h	2022-09-24 06:27:24.035810356 +0000
+@@ -217,6 +217,7 @@
+ 	struct timecounter tc;
+ 	struct timespec64 prev_ptp_time; /* Pre-reset PTP clock */
+ 	ktime_t ptp_reset_start; /* Reset time in clock mono */
++	char fw_version[32];
+ };
+ 
+ void igc_up(struct igc_adapter *adapter);
+--- a/igc_base.c	2022-09-23 12:17:01.000000000 +0000
++++ b/igc_base.c	2022-09-24 07:07:24.414598890 +0000
+@@ -205,6 +205,7 @@
+ 	case IGC_DEV_ID_I220_V:
+ 	case IGC_DEV_ID_I225_K:
+ 	case IGC_DEV_ID_I225_K2:
++	case IGC_DEV_ID_I226_K:
```

Comment:
What is the goal of this addition?

Reply:
To support the new I226 which is just the I225 v4 with a different ID. I have a new card on the way for empirical testing. Reviewing Intel's code post there is at least one more addition to make.